### ISSUE TITLE
:cd not working as expected ..?

### ISSUE THREAD

**Issue by tayloraswift** [NeoVintageous/NeoVintageous#797]
**Describe the bug**  
When typing :pwd, it shows that I'm in the right working directory, but upon cd-ing into anything lower from there it thinks it can't find it because then it considers C:/Program Files/Sublime Text to be the present working directory ..? Or I might just be getting this all wrong, I'm not too sure :(

**To Reproduce**  
Steps to reproduce the behaviour:
1) Open up a folder (let's call it folder A) which has a higher level folder (let's call it folder B) that you can cd into (not sure if the wording here is right but I hope it still makes sense).
2) Since you opened up folder A in Sublime, NeoVintageous should think folder A is your present working directory, so type :pwd to verify that you're currently in folder A.
3) Try to cd into folder B and (if you have the same setup as I do with Sublime text is Program Files) it should tell you that 'Can't find directory "C:/Program Files/Sublime Text/[folder B name]" in cdpath'
4) If you tried this in command line Vim it'd work and set your present working directory to be folder B.

**Expected behavior**  
Your present working directory should be folder B after :cd [folder B name]

**Actual behavior**  
You get an error !!

**Versions**  
Show the Sublime Text console log: `Menu > View > Show Console`. 
Version 4121 Stable

**Additional Information**  
There's not really much else, again I could be just really confused about how this all works and if so, I'm so sorry this can be closed. I'm also wondering how / why "set autochdir" in the rc file doesn't seem to work. Even if I change around files, the dir does not seem to change with me :(

**Comment by gerardroche** [comment]
`set autochdir` is not currently supported.

When you say _open up a folder_ can you show me how you're opening it? Also when you say _cd into a folder_ how are you cd-ing into the folder`, via `:cd ../`?

But yes, I've done various testing and found some unexpected behaviours.

**Comment by tayloraswift** [comment]
Ah, I've actually just realized that the way I open up folders is not very common for ST. I've edited my Windows registry so that I'm able to open folders by right clicking them and pressing "Open in Sublime Text", perhaps that's why it behaves weirdly.
![image](https://user-images.githubusercontent.com/58350457/145597432-5b327349-7ef9-430e-884e-319c3ea4adb1.png)

When I say "cd into a folder" I mean that after having opened it as above, to get into a folder within that I go :cd [folder within's name], so if I had folder B within folder A, after having opened A, I'd run :cd B which wouldn't work, but perhaps that's because of my unconventional way for opening folders. Thank you very much though! I guess I'll just have to live with it.
